melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/sed/mapper/SedMapper.java

```diff
@@ -171,6 +172,9 @@ default void setFamiliemedlemmer(SedDataDto sedData, Bruker bruker) {
     }
 
     default List<Arbeidssted> hentArbeidssted(SedDataDto sedData) throws MappingException, NotFoundException {
+        if (CollectionUtils.isEmpty(sedData.getArbeidssteder())) {
+            return List.of(tomtArbeidssted());
+        }
```

Comment:
Dette blir vel feil? Hvis man bare setter `erIkkeFastAdresse`, så får vi en valideringsfeil, for da er felt 7.1.1 og 7.1.3 påkrevd?

Reply:
Ah, det tenkte jeg ikke over... Skal ta en prat med Jonas.

Reply:
Er bare å si det som det er når det er Rina-krav:
DU FÅR DET IKKE SOM DU VIL.